cdist/conf/type/__staged_file/gencode-local

```diff
@@ -62,7 +61,7 @@ fetch_file() {
 }
 
 fetch_and_prepare_file() {
-   printf 'tmpdir="$(mktemp -d --tmpdir="/tmp" "%s")"\n' "${__type##*/}.XXXXXXXXXX"
+   printf 'tmpdir="$(TMPDIR="/tmp" mktemp -d "%s")"\n' "${__type##*/}.XXXXXXXXXX"
```

Comment:
TMPDIR might be set by the user. `TMPDIR=${TMPDIR:-/tmp}` might be more appropriate here.